src/confluence/io/json.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import pandas as pd
+from collections import OrderedDict
+import json
+import os
+_logger = logging.getLogger(__name__)
+
+
+def read(fname, **kwds):
+    """
+    Reads a JSON-formatted filename into a Container.
+    Args:
+        fname: File name or file object.
+        **kwds: Keywords handled by pandas.read_excel
+
+    Returns:
+        Container object.
+    """
+    try:
+        with open(fname) as json_file:
+            data = json.load(json_file)
+        _logger.debug(f"Reading JSON file {fname}")
+        dfod = {}
+        try:
+            for k, v in data.items():
+                _logger.debug(f"Adding sheet {k} to 'dfod'")
```

Comment:
Don't use abbreviations here, i.e. `_logger.debug(f"Adding sheet {k} to OrderedDictionary of DataFrames (dfod)."`